exif2timestream.py

```diff
@@ -236,18 +249,111 @@ def __call__(self, x):
         return None
 
 
+def resize_img(filename, to_width):
+    # Open the Image and get its width
+    if not(SKIMAGE):
+        warnings.warn(
+            "Skimage Not Installed, Unable to Test Resize", ImportWarning)
+        return None
```

Comment:
Don't bother changing this, but in future, `return` and `return None` are identical.